client-web/src/functional-e2e/authentication/check.spec.ts

```
@@ -0,0 +1,47 @@
+test.beforeAll(async () => {
+  const a = await TestScenarioFactory.createBaseScenario(scenarioConfig);
+  console.log('Scenario created:', a);
+});
+
+test.beforeEach(async ({ context }) => {
+  await context.clearCookies();
+  await deleteMailSlurperMails();
+});
+test.describe.configure({ mode: 'serial' });
+
+test.only('verify registration page', async ({ page }) => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)